<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuditServeurMultiWidget</class>
 <widget class="QWidget" name="AuditServeurMultiWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AuditServeurMonoWidget</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>264</width>
     <height>310</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_numPort">
          <property name="text">
           <string>Num√©ro de port :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_port">
          <property name="maximum">
           <number>10000</number>
          </property>
          <property name="value">
           <number>8888</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton_lancement">
          <property name="text">
           <string>Lancement serveur</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Informations</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="textEdit_Information"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Quitter">
      <property name="text">
       <string>Quitter</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Quitter</sender>
   <signal>clicked()</signal>
   <receiver>AuditServeurMultiWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>427</y>
    </hint>
    <hint type="destinationlabel">
     <x>708</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
